components/util/parsely.coffee

```diff
@@ -0,0 +1,20 @@
+request = require 'superagent'
+{ reject } = require 'underscore'
+
+module.exports =
+
+  topParselyArticles: (section, articleSlug, key, secret, cb) ->
```

Comment:
This seems small enough to just be a method to add to the Article or Articles model/collection (even though it doesn't return _that_ model data)—that might also help DRY up some of that router fetching I noted above by co-locating that code.